widget.json

```diff
@@ -0,0 +1,1269 @@
+{
+  "name": "Slider",
+  "package": "com.fliplet.slider-container",
+  "version": "1.0.0",
+  "icon": "img/icon.png",
+  "tags": [
+    "type:component",
+    "category:general"
+  ],
+  "provider_only": false,
+  "references": [
+    "slides:richContent",
+    "redirectEndScreen:page"
+  ],
+  "settings": {
+    "asyncRendering": true,
+    "interfaceWidgetPackage": "com.fliplet.helper-configuration"
+  },
+  "html_tag": "div",
+  "interface": {
+    "dependencies": [
```

Comment:
I don't think these dependencies are needed if you've already configured it to use an interface widget package above.

```suggestion
```